xbmc/platform/posix/filesystem/SMBFile.h

```diff
@@ -79,7 +79,7 @@ class CSMBFile : public IFile
   bool OpenForWrite(const CURL& url, bool bOverWrite = false) override;
   bool Delete(const CURL& url) override;
   bool Rename(const CURL& url, const CURL& urlnew) override;
-  int GetChunkSize() override { return 64*1024; }
+  int GetChunkSize() override { return 128 * 1024; }
```

Comment:
Are 100% sure 128K chunks won't cause any regressions? 64K has been around for ages and known to work....

Reply:
> Nothing is certain but my reasoning is that 128K was already used with NFSv3 and the file types are the same. 128K is also used on Windows with FileCache: i.e. when size is zero or < 1 FileCache forces to use 128K chunk size (default in advanced settings).
> 
> Maybe we can merge it but not backport to 20.1 but wait until 20.2. Or backport only Windows commit as it fixes a major bug.
> 

Sounds good.